dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.h

```diff
@@ -625,6 +625,7 @@ class ModelBus : public NetworkComponent::Impl {  ///< Generic AC network bus
   std::vector<boost::weak_ptr<ModelSwitch> > connectableSwitches_;  ///< switch connected or connectable on the node
 
   const std::string modelType_;  ///< model Type
+  const bool isNodeBreaker_;  ///< true if bus is a calculated one
```

Comment:
I would suggest the following comment: true if the bus is in node breaker (= calculated bus) or vice versa.

Reply:
ok, thanks